core/server/worker/src/test/java/alluxio/worker/netty/BlockReadHandlerTest.java

```diff
@@ -25,7 +25,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mockito;
+import static org.mockito.Mockito;
```

Comment:
see https://github.com/Alluxio/alluxio/pull/7057/files for an example of how to use static imports